Datadog.Trace.proj

```diff
@@ -17,7 +17,8 @@
     <FrameworkReproduction Include="reproductions\EntityFramework6x.MdTokenLookupFailure\EntityFramework6x.MdTokenLookupFailure.csproj" />
   </ItemGroup>
 
-  <Import Project="PackageVersions.g.props" />
+  <Import Condition="'$(PerformExhaustiveTesting)'=='true'" Project="PackageVersionsExhaustive.g.props" />
```

Comment:
@bobuva This `PerformExhaustiveTesting` will run for every version. We just need to hook it into CI.